pkg/jobs/jobspb/jobs.proto

```diff
@@ -695,40 +695,53 @@ message SchemaChangeGCProgress {
   bool ranges_unsplit_done = 4;
 }
 
-message ChangefeedTarget {
+message ChangefeedTargetTable {
```

Comment:
I'm curious: why do we need to change ChangefeedTarget and introduce 2 new types? Couldn't we extend
ChangefeedTarget message itself?  Say, add the TargetType (defulat == 0 -- so , same as now), add e.g. family_name if target type != 0?

Reply:
My reasoning here is that we won't be able to put things like family_name into the ChangefeedTargets (note plural) proto, since it's an ID -> Target map and column families don't have globally unique ids, and the map needs to stay for now for inter-version compatibility. So if we extend ChangefeedTarget, we end up with both a map and an array of ChangefeedTarget where the copies in the map are missing some data, which seems overall more confusing. We don't technically have to change ChangefeedTarget to ChangefeedTargetTable, I just wanted the name to stay accurate.